API doc for perpetual account lockout prevention (#3791)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/users/index.md

| Parameter | Description                                                        | Param Type | DataType | Required |
| --------- | ------------------------------------------------------------------ | ---------- | -------- | -------- |
| id        | `id`, `login`, or `login shortname` (as long as it is unambiguous) | URL        | String   | TRUE     |

> | expand        | Valid value: `block`. If you have migrated to Okta Identity Engine, you can specify `blocks` to include account block details in the `_embedded` attribute. The embedded object will be the same as the one returned by the [Get User's Blocks](#get-user-s-blocks) operation | Query        | String   | FALSE     |

Reviewer: ```suggestion
| expand  <ApiLifecycle access="ie" />  | Valid value: `block`. If this parameter is specified, then account block details are included in the `_embedded` attribute. The returned embedded object is the same as the one returned by the [Get User Blocks](#get-user-blocks) operation. | Query        | String   | FALSE     |
```

]
```

> ### Get User's Blocks

Reviewer: ```suggestion
### Get User Blocks
```

<ApiOperation method="get" url="/api/v1/users/${userId}/blocks" />

Lists information about how the account is blocked from access

##### Request parameters

| Parameter | Description  | Param Type | DataType | Required |
| --------- | ------------ | ---------- | -------- | -------- |

> | id        | `id`, `login`, or `login shortname` (as long as it is unambiguous) of user | URL        | String   | TRUE     |

Reviewer: ```suggestion
| id        | `id`, `login`, or `login shortname` (as long as it is unambiguous) of the user | URL        | String   | TRUE     |
```

}
```

#### User Block object

> The object describes how the account is blocked from access. If `appliesTo` is `ANY_DEVICES`, the account is blocked for all devices. If `appliesTo` is `UNKNOWN_DEVICES`, the account is only [blocked for unknown devices](/docs/reference/api/attack-protection/#user-lockout-settings-object).

Reviewer: ```suggestion
The User Block object describes how the account is blocked from access. If `appliesTo` is `ANY_DEVICES`, then the account is blocked for all devices. If `appliesTo` is `UNKNOWN_DEVICES`, then the account is only [blocked for unknown devices](/docs/reference/api/attack-protection/#user-lockout-settings-object).
```

| Property  | Description | Datatype |
| :---------| :---------- | :------- |

> | type      | Type of the block. Valid values: `DEVICE_BASED` | String |

Reviewer: ```suggestion
| type      | Type of the block. Valid value: `DEVICE_BASED` | String |
```
